tests/Mockery/NamedMockTest.php

```diff
@@ -51,4 +53,16 @@ public function itShouldThrowIfAttemptingToRedefineNamedMock()
         $mock = Mockery::namedMock("Mockery\Dave7");
         $mock = Mockery::namedMock("Mockery\Dave7", "DateTime");
     }
+
+    /** @test */
+    public function itCreatesConcreteMethodImplementationWithReturnType()
+    {
+        $cactus = new \Nature\Plant();
+        $gardener = Mockery::namedMock(
+            "NewNamespace\\ClassName",
+            Gardener::class,
```

Comment:
I think the `class` constant is failing the tests, it looks like we're still supporting PHP 5.4. We should probably drop it, but for the sake of getting this merged you may as well make the change here.